src/operator/bufferCount.ts

```diff
@@ -83,20 +121,21 @@ class BufferCountSubscriber<T> extends Subscriber<T> {
       buffer.push(value);
       if (buffer.length === bufferSize) {
         buffers.splice(i, 1);
-        destination.next(buffer);
+        this.destination.next(buffer);
       }
     }
   }
 
-  protected _complete() {
-    const destination = this.destination;
+  protected _complete(): void {
     const buffers = this.buffers;
+
     while (buffers.length > 0) {
       let buffer = buffers.shift();
       if (buffer.length > 0) {
-        destination.next(buffer);
+        this.destination.next(buffer);
```

Comment:
`destination` lookup used to be cached above, so it doesn't need to happen every time. Seemed like a simple win for micro perf?